frontend/src/metabase/admin/tools/components/Logs/utils.ts

```diff
@@ -77,11 +77,10 @@ export function filterLogs(logs: Log[], { process, query }: UrlState) {
   const lowerCaseQuery = query ? query.toLowerCase() : "";
 
   return logs.filter((log) => {
-    const formattedLog = formatLog(log).join("\n");
-
     const matchesProcessFilter =
       !process || process === "ALL" || log.process_uuid === process;
-    const matchesQueryFilter = formattedLog
+    // search for text in any of log's properties without expensive formatting or parsing
+    const matchesQueryFilter = JSON.stringify(log)
```

Comment:
I like the impetus to bypass the expensive formatting, but now we've got a situation where we're potentially searching data that's not visible, like the instance UUID